.github/workflows/ci.yml

```diff
@@ -57,35 +60,54 @@ jobs:
           echo "  github.event.ref: ${{ github.event.ref }}"
           echo "---"
           echo "concurrency-group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}"
+      - name: labeler
+        id: labeler
+        if: github.event_name == 'pull_request'
+        uses: lablup/auto-labeler@main   # actions/labeler, lablup/size-label-action, lablup/auto-label-in-issue
       - name: Optimize CI
-        id: check_skip
+        id: check_stacked_pr
         run: |
-          echo "SKIP=false" >> $GITHUB_OUTPUT
+          echo "NON_TOP_BOTTOM=false" >> $GITHUB_OUTPUT
           BASE_BRANCH=${{ github.event.pull_request.base.ref }}
           if [[ "$BASE_BRANCH" != "main" && ! "$BASE_BRANCH" =~ ^[0-9]{2}\.[0-9]{1,2}$ ]]; then
             response=$(curl -s -o /dev/null -w "%{http_code}" -X GET "${{ secrets.KVSTORE_URL }}/?key=base_${{ github.event.pull_request.head }}" \
             -H "Authorization: Bearer ${{ secrets.KVSTORE_TOKEN }}")
             if [ "$response" == "200" ]; then
-              echo "SKIP=true" >> $GITHUB_OUTPUT
+              echo "NON_TOP_BOTTOM=true" >> $GITHUB_OUTPUT
             fi
           fi
-      - name: labeler
-        if: github.event.action == 'opened' || github.event.action == 'reopened' || github.event.action == 'synchronize'
-        uses: lablup/auto-labeler@main   # actions/labeler, lablup/size-label-action, lablup/auto-label-in-issue
+      - name: Skip CI
+        id: check_skip
+        if: |
+          contains(toJSON(github.event.head_commit.message), 'skip:ci')
+          || contains(github.event.pull_request.labels.*.name, 'skip:ci')
+          || steps.check_stacked_pr.outputs.NON_TOP_BOTTOM == 'true'
+        run: echo "SKIP_CI=true" >> $GITHUB_OUTPUT
+      - name: Graphite labels
+        id: graphite_labels
+        if: contains(fromJSON('["flow:merge-queue", "flow:hotfix"]'), github.event.pull_request.labels.*.name)
+        run: |
+          echo "GRAPHITE_LABELS=true" >> $GITHUB_OUTPUT
+      - name: Release
+        id: release
+        if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+        run: echo "IS_RELEASE=true" >> $GITHUB_OUTPUT
+      - name: Alembic
+        id: alembic
+        if: contains(steps.labeler.outputs.all-labels, 'require:db-migration')
+        run: echo "ALEMBIC=true" >> $GITHUB_OUTPUT
```

Comment:
`GRAPHITE_LABELS` and `ALEMBIC` seem ambiguous. Could you make them more explicit?